Jenkinsfile

```diff
@@ -63,12 +114,13 @@ pipeline {
                         }
                     }
                     steps {
-                        runTest script: '''set -ex
-                                           rm -f *.xml
-                                           echo "<failure> bla bla bla</failure>" > \
-                                             pipeline-test-failure.xml''',
-                            junit_files: '*.xml non-exist*.xml',
-                            failure_artifacts: env.STAGE_NAME
+                        job_step_update(
+                            runTest(script: '''set -ex
```

Comment:
Do these **need** the brackets?  We have been using a coding style where we treat these as _step_ names rather than function names in `Jenkinsfile`, and only treat them as function names in pure groovy scripts.